guides/V1_space_template_info.md

```diff
@@ -0,0 +1,93 @@
+---
+copyright: 'Copyright IBM Corp. 2018'
+link: 'space-template-info'
+is: 'experimental'
+---
+
+# Space Template Info
+
+In addition to those space properties documented publicly, you can add _templateInfo_ to your _space_ request in order to fetch
+information about the template used to create a space.
+
+## Schema
+
+### Get Template Info
+
+_templateInfo_ is a property of a _Space_ object which is exposed when you add `EXPERIMENTAL` to the `x-graphql-view` header.
+
+```graphql
+type Space {
+...
+  templateInfo: SpaceTemplateInfo
+}
+```
+
+The _SpaceTemplateInfo_ object comprises a subset of the fields in the _SpaceTemplate_ type.
+
+```graphql
+type SpaceTemplateInfo {
+  id: ID!
+  name: String!
+  description: String!
+  spaceStatus: SpaceStatus!
+  requiredApps: SpaceRequiredApplicationCollection
+  properties: SpacePropertyCollection
+  labelIds: [String]
+  created: Date
+  createdBy: Person
+  updated: Date
+  updatedBy: Person
+}
+```
+
+## Example request
+
+~~~~
+Method: POST
+URL: https://api.watsonwork.ibm.com/graphql
+Headers: 'Content-Type: application/graphql' , 'x-graphql-view: PUBLIC, EXPERIMENTAL'
+Body:
+{
+  space(id: "space-id") {
```

Comment:
I already think this is getting very costly to maintain with the same data copy-pasted across files. 

I think we should look at some template engine if we need to have this information on all pages. For now I suggest we just link to the explorer main page.

+```graphql
+type Space {
+...
+  templateInfo: SpaceTemplateInfo
+}
+```
I think this should be enough for anyone somewhat familiar with GraphQL.

Reply:
We could have a generic example for posting a query instead of one for each query.

Reply:
Removed the example request and replaced with a link to explorer main page.